hack/ginkgo-e2e.sh

```diff
@@ -81,7 +81,10 @@ if [[ -n "${CONFORMANCE_TEST_SKIP_REGEX:-}" ]]; then
   ginkgo_args+=("--skip=${CONFORMANCE_TEST_SKIP_REGEX}")
   ginkgo_args+=("--seed=1436380640")
 fi
-if [[ ${GINKGO_PARALLEL} =~ ^[yY]$ ]]; then
+if [[ -n "${GINKGO_PARALLEL_NODES:-}" ]]; then
+  ginkgo_args+=("--nodes=${GINKGO_PARALLEL_NODES}")
```

Comment:
@ixdy So by default. -p will sets the default number. From the ginkgo doc:

>The number of nodes used with -p is runtime.NumCPU() if runtime.NumCPU() <= 4, otherwise it is runtime.NumCPU() - 1 based on a rigorous science based heuristic best characterized as “my gut sense based on a few months of experience”